kickoff/static/model/release.js

```diff
@@ -1,37 +1,78 @@
+// Please also update VALID_VERSION_PATTERN in forms.py
+// TODO: Read the pattern from a centralized file
+var VALID_VERSION_PATTERN_STRING  = '^(\\d+)\\.(' + // Major version number
```

Comment:
Oh, this is duplicating code... What about creating a library for this? I am sure we would have other places to use it.